RELEASE/data/autoscend_properties.txt

```diff
@@ -0,0 +1,225 @@
+0	auto_hpAutoRecoveryItems
```

Comment:
this is all the get/set_property properties used throughout autoscend. Used for the migration function right now, would be nice to start maintaining it with descriptions, type and defaults

Reply:
Nice! It's helpful to have an authoritative list, and I've often wished that properties had comments. Personally I'd rather them be code comments, but no big deal.